doc/sphinxman/source/introduction.rst

```diff
@@ -93,6 +93,19 @@ Overall |PSIfour| Package
 The following citation should be used in any publication utilizing the
 |PSIfour| program package:
 
+* "Psi4 1.1: An Open-Source Electronic Structure Program Emphasizing
+  Automation, Advanced Libraries, and Interoperability", R. M. Parrish, L.
```

Comment:
I haven't checked how this renders, but does the break in "L. A. Burns" cause it to be split across two lines?  If so, you probably want a line break before the L.  LaTeX allows things to be forced onto the same line with non-breaking spaces (~), but I'm not aware of a simlar feature in REST.